src/styles/blocks/timer.scss

```diff
@@ -0,0 +1,52 @@
+.timer {
```

Comment:
Ah you chicky,  my first thought was like: "wow he did everything with html and css"